integrations/make.com/integration.definition.ts

```diff
@@ -0,0 +1,35 @@
+import { IntegrationDefinition, messages } from '@botpress/sdk';
+import { z } from 'zod';
+
+const INTEGRATION_NAME = "makecom";
+
+export default new IntegrationDefinition({
+  name: INTEGRATION_NAME,
+  version: '0.2.0', 
+  icon: 'icon.svg', 
+  readme: 'hub.md',
+  configuration: {
+    schema: z.object({
+      webhookUrl: z.string().url().describe('Make.com webhook URL'),
+    }).describe('Configuration schema for Make.com Integration'),
+  },
+  channels: {
+    channel: {
+      messages: { ...messages.defaults },
+    },
+  },
+  actions: {
+    sendData: {
+      input: {
+        schema: z.object({
+          data: z.string().min(1, { message: "Must not me empty"}).describe("JSON string of data to send")
```

Comment:
**chore(blocking):** Make sure to run [prettier](https://prettier.io/) on your code, the prettier config is already at the root of the repository, it should automatically load it if you already have it installed. If you need help running it, feel free to reach out to me directly on Discord!